components/dynamic/SummitData/Weather/Current/index.js

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useWeatherUnit } from "@/contexts/WeatherUnit";
+import { useSummitData } from "@/contexts/SummitData";
+import Loader from "@/components/atomic/Loader";
+import WidgetSection from "@/components/layout/WidgetSection";
+import TemperatureCurrent from "@/components/widgets/CurrentData/patterns/Temperature";
+import WindspeedCurrent from "@/components/widgets/CurrentData/patterns/Windspeed";
+import PrecipitationCurrent from "@/components/widgets/CurrentData/patterns/Precipitation";
+import { convertTemperature, convertWindspeed } from "@/helpers/converters";
+
+const CurrentWeather = () => {
+  const { t } = useTranslation();
+  const [isOpen, setOpen] = useState(true);
+  const [{ tempUnit, windspeedUnit }] = useWeatherUnit();
+  const { currentData, loading = true } = useSummitData();
+  const sectionProps = {
+    title: t("summit_dashboard.sections.weather.current"),
+    onToggleCallback: (value) => setOpen(value),
+    isOpen,
+  };
+
+  if (loading || !currentData)
```

Comment:
super rudimentary undefined protection, it'll just stick in loading state forever. We should consult José on what kind of error status to put the UI in. Since we're going to be calling from a bunch of data sources I think having a robust error handling solution would be wise